### DISCUSSION TITLE
怎么指向自己的ip

### DISCUSSION THREAD

**Discussion by superen7527** [XIU2/CloudflareSpeedTest#461] [讨论]
如果我想让域名自动解析不改本地，应该是用ddns 看了3个小时项目讨论发现两个办法

1 付费域名放CF 用CFapi改解析
2 免费域名放dnspod api改解析然后CFcname

问题 假设域名github.com在CF如果我要z.github.com指向我的ip1
根据上面两个用法
1 根据https://github.com/XIU2/CloudflareSpeedTest/issues/40没看懂那么不开小云朵让他指向优选ip 那怎么指向ip1 不想用saas
2 如果放腾讯云作者说可以cname不会1000那应该用z.github.com指向cname免费域名 还是不能回到ip1啊

**Comment by XIU2** [comment]
如果你是做网站的话，那么 Cloudflare 上你只能 SaaS 接入去自选。
如果你是做代理什么的话，那么就直接按照 #40 来更新域名解析 IP 即可。

**Reply by superen7527** [reply]
不玩代理 那种滥用迟早把CF玩没

我是做网站的 不能只有我一个人快 如果回源域名不在CF有点不方便而且证书还要手动 唉

**Reply by superen7527** [reply]
想了一个解决方案 回源能不能用api实现 这样就可以自动续费证书 然后用别的域名回源优选 我最后直接cname过去 应该也可以吧
